modules/examples/flake/flake.nix

```diff
@@ -0,0 +1,47 @@
+{
+  description = "Example darwin system flake";
+
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs";
+    darwin.url = "github:lnl7/nix-darwin";
+    darwin.inputs.nixpkgs.follows = "nixpkgs";
+  };
+
+  outputs = { self, darwin, nixpkgs }:
+  let
+    configuration = { pkgs, ... }: {
+      # List packages installed in system profile. To search by name, run:
+      # $ nix-env -qaP | grep wget
+      environment.systemPackages =
+        [ pkgs.vim
+        ];
+
+      # Auto upgrade nix package and the daemon service.
+      # FIXME: for github actions, this shouldn't be in the example.
```

Comment:
Users will see this after running init against this template, and I feel there is not enough context provided.